internal/cmdutils/login/login.go

```diff
@@ -27,10 +27,11 @@ func GetToken(server string) string {
 }
 
 func ReadTokenInteractively(server string) (string, error) {
-	url, err := url.JoinPath(server, "/dashboard/settings/account/tokens?create&origin=cli")
+	url, err := url.JoinPath(server, "/dashboard/settings/account/tokens")
```

Comment:
```suggestion
	url, err := url.JoinPath(server, "dashboard", "settings", "account", "tokens")
```
I'm not sure but wouldn't this make more sense than to write the slashes manually?